deploy/backup.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: ps.percona.com/v1alpha1
 kind: PerconaServerMySQLBackup
 metadata:
   name: backup1
+  finalizers:
```

Comment:
please comment it by default

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/158/commits/c5333d99b8176fd5a69c8fdb6b927ae0f585f5b3